src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(operation, value) {
+      this.result = operation(this.result, value);
+
+      return this;
+    },
+
+    add(a, b) {
```

Comment:
```suggestion
    add(value) {
    this.result += value
```
that way calculator will work correctly